how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -99,115 +122,132 @@ $ services=(
 
-	cat << EOT | sudo tee "${service_dir}/proxy.conf"
+    cat << EOT | sudo tee "${service_dir}/proxy.conf"
 [Service]
 Environment="HTTP_PROXY=${http_proxy}"
 Environment="HTTPS_PROXY=${https_proxy}"
 Environment="NO_PROXY=${no_proxy}"
 EOT
 done
+
 $ sudo systemctl daemon-reload
 ```
 
-###  Start Kubernetes with kubeadm 
+## Start Kubernetes
 
-```bash
-# Mark sure containerd is up and running
-$ sudo systemctl restart containerd
-$ sudo systemctl status containerd
+- Make sure `containerd` is up and running
 
-# Prevent docker iptables rules conflict with k8s pod communication
-$ sudo iptables -P FORWARD ACCEPT
+  ```bash
+  $ sudo systemctl restart containerd
+  $ sudo systemctl status containerd
+  ```
 
-# Start cluster using kubeadm
-$ sudo kubeadm init --skip-preflight-checks \
---cri-socket /run/containerd/containerd.sock --pod-network-cidr=10.244.0.0/16
+- Prevent conflicts between `docker` iptables (packet filtering) rules and k8s pod communication
 
-$ export KUBECONFIG=/etc/kubernetes/admin.conf
+  ```bash
+  $ sudo iptables -P FORWARD ACCEPT
+  ```
 
-$ sudo -E kubectl get nodes
-$ sudo -E kubectl get pods
-```
+- Start cluster using `kubeadm`
 
-### Install a pod network
-Install a pod network plugin is needed to allow pods communicate with each other.
+  ```bash
+  $ sudo kubeadm init --skip-preflight-checks --cri-socket /run/containerd/containerd.sock --pod-network-cidr=10.244.0.0/16
+  $ export KUBECONFIG=/etc/kubernetes/admin.conf
+  $ sudo -E kubectl get nodes
+  $ sudo -E kubectl get pods
+  ```
 
-Install flannel plugging, by following the instructions in the section *Installing a pod network*
-from [Using kubeadm to Create a Cluster ](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/)
-guide.
+## Install a Pod Network
 
-<!---
-```bash
-# Install a pod network using flannel
-# There is not a programmatic way to know last what flannel commit use
-# See https://github.com/coreos/flannel/issues/995
-$ sudo -E kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
-```
---->
+A pod network plugin is needed to allow pods to communicate with each other.
 
+- Install the `flannel` plugin by following the
+  [Using kubeadm to Create a Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#instructions)
+  guide, starting from the **Installing a pod network** section.
 
-```bash
-# wait for pod network
-$ timeout_dns=0
-$ until [ "$timeout_dns" -eq "420" ]; do
-	if sudo -E kubectl get pods --all-namespaces | grep dns | grep Running; then
-		break
-	fi
-	sleep 1s
-	((timeout_dns+=1))
- done
-
-# check pod network is running
-$ sudo -E kubectl get pods --all-namespaces | grep dns | grep Running && echo "OK" || ( echo "FAIL" && false )
-```
+- Create a pod network using flannel
 
-### Allow run pods in master node
+  **Note:** There is not a programmatic way to know last what flannel commit use. See https://github.com/coreos/flannel/issues/995.
```

Comment:
s/not a/no/

Reply:
Yoda speak I read this: `know last what flannel commit use` (and how can there be no :yoda: emoji!) ;-)